eslint.config.mjs

```diff
@@ -35,6 +36,7 @@ export default [
   {
     name: 'dd-trace/global-ignore',
     ignores: [
+      '**/.bun', // Ignore autogenerated bun files
```

Comment:
This is an independent fix. Running the linter with these entries there fails the linter and that makes it hard locally.